brycemara/overlook
# Feature/login form

## What is the change?
Add's HTML for the Login Display, builds out the webpack.
## Is this a fix or a feature? 
Feature.
## Where should the reviewer start?
Line 1 of index.html and base.scss.
## How should this be tested?
Make sure the login display is coming up with no errors in the console.
_There's no functionality to the login form yet_
## Any relevant tickets?
Ticket #20 
## Screenshots (optional)
![Screen Shot 2020-11-04 at 10 18 24 AM](https://user-images.githubusercontent.com/68242949/98137959-36e07f80-1e88-11eb-91b0-adb88346079c.png)

@brycemara
